modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -0,0 +1,276 @@
+[id="cluster-logging-collector-log-forward-cloudwatch_{context}"]
+= Forwarding logs to Amazon CloudWatch
+
+You can forward logs to Amazon CloudWatch, a monitoring and log storage service hosted by Amazon Web Services (AWS). You can forward logs to CloudWatch in addition to, or instead of, the default OpenShift Logging-managed Elasticsearch log store.
+
+To configure log forwarding to CloudWatch, create a `ClusterLogForwarder` custom resource (CR) with an output for CloudWatch and a pipeline that uses the output.
+
+.Procedure
+
+. Create a `Secret` YAML file that uses the `aws_access_key_id` and `aws_secret_access_key` fields to specify your base64-encoded AWS credentials. For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cw-secret
+  namespace: openshift-logging
+data:
+  aws_access_key_id: QUtJQUlPU0ZPRE5ON0VYQU1QTEUK
+  aws_secret_access_key: d0phbHJYVXRuRkVNSS9LN01ERU5HL2JQeFJmaUNZRVhBTVBMRUtFWQo=
+----
+
+
+. Create the secret. For example:
++
+[source,terminal]
+----
+$ oc apply -f cw-secret.yaml
+----
+
+
+. Create a `ClusterLogForwarder` CR YAML file that specifies the name of the secret. For example:
++
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: instance <1>
+  namespace: openshift-logging <2>
+spec:
+  outputs:
+   - name: cw <3>
+     type: cloudwatch <4>
+     cloudwatch:
+       groupBy: logType <5>
+       region: us-east-2 <6>
+     secret:
+        name: cw-secret <7>
+  pipelines:
+    - name: infra-logs <8>
+      inputRefs: <9>
+        - infrastructure
+        - audit
+        - application
+      outputRefs:
+        - cw <10>
+----
+<1> The name of the `ClusterLogForwarder` CR must be `instance`.
+<2> The namespace for the `ClusterLogForwarder` CR must be `openshift-logging`.
+<3> Specify a name for the output.
+<4> Specify the `cloudwatch` type.
+<5> Optional: Specify the option CloudWatch uses to group the logs. If you omit `groupBy`, its default value is `logType`.
+<6> Specify the AWS region.
+<7> Specify the name of the secret that contains your AWS credentials, which you created earlier.
```

Comment:
Consider saying which step the user created the credential. Ex: `...that contains your AWS credentials, which you created in <name of step>` 

Also, I am reading this as `the secret that you created earlier`, so Step 2. However, do you mean, `the AWS container that you created earlier`? 

I think how it is written grammatically, the phrase `which you created earlier` would have to modify `credentials` **not** `secret`

Reply:
@jc-berger We're assuming the person configuring forwarding of logs to CloudWatch is familiar with AWS credentials and are not covering that part. There's plenty of AWS docs on how to do that. We're not creating an AWS container, either. We're asking the user to base64-encode their AWS credentials, put them into a secret and give us the name of that secret in the ClusterLogForwarder.

Reply:
Thanks. Updated.